app/assets/stylesheets/invitations.sass

```diff
@@ -5,11 +5,61 @@
   padding: $form-padding
   margin: $form-margin
   input[type="text"]
-    width: 100%
-    padding: 10px
-    margin: 0 0 10px
-    border-radius: 5px
     font-size: $screensmart-font-size
+    box-sizing: border-box
+    width: 100%
+    height: calc(#{$form-text-input-height} + #{$form-text-input-borders})
+    margin: 0.5rem 0 $form-text-input-margin
+    padding: $form-text-input-padding
+    border: $form-text-input-border solid #ddd
+    border-radius: $option-border-radius
+    border-style: inset
+    background: $form-background-color
+    resize: none
+    outline: none
+    &[required]
```

Comment:
Het is volgens mij semantisch incorrect om dit alleen te doen voor required velden. Waarom zou je deze styling niet willen bij niet-required velden?